packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -2506,6 +2508,15 @@ class DiagnosticsProperty<T> extends DiagnosticsNode {
   /// of the property should be displayed without line breaks if possible.
   String valueToString({ TextTreeConfiguration parentConfiguration }) {
     final T v = value;
+    // Normalize web/DDC specific closure toString syntax.
```

Comment:
This change is unrelated to the PR description, no?

Reply:
Not sure I understand the motivation behind this?

Reply:
Oh woops, yes this one is slightly unrelated.